README.md

```diff
@@ -1,24 +1,67 @@
-# README
+# Project: Private Events!
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-* Ruby version
+build a site similar to a private Eventbrite which allows users to create events and then manage user signups. 
 
-* System dependencies
+https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/associations
 
-* Configuration
+A user can create events. A user can attend many events. An event can be attended by many users. 
 
-* Database creation
 
-* Database initialization
+## Getting started
 
-* How to run the test suite
+To get started with the app, first clone the repo and `cd` into the directory:
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+$ git clone https://github.com/od-c0d3r/private-events.git
+$ cd private-events
+```
 
-* Deployment instructions
+Then install the needed packages (while skipping any Ruby gems needed only in production):
 
-* ...
+```
+$ bundle install
+```
+
+Next, migrate the database:
+
+```
+$ rails db:migrate
+```
```

Comment:
[OPTIONAL] You can ask the user to use command `rails s `after migration to open your app.